tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -36,164 +31,91 @@ class CentralNode(SKABaseDevice):
-                    elif (self._dish_mode == 1):
-                        print "Dish Mode :-> STARTUP"
-                    elif (self._dish_mode == 2):
-                        print "Dish Mode :->  SHUTDOWN"
-                    elif (self._dish_mode == 3):
-                        print "Dish Mode :->  STANDBY-LP"
-                    elif (self._dish_mode == 4):
-                        print "Dish Mode :-> STANDBY-FP"
-                    elif (self._dish_mode == 5):
-                        print "Dish Mode :-> MAINTENANCE"
-                    elif (self._dish_mode == 6):
-                        print "Dish Mode :-> STOW"
-                    elif (self._dish_mode == 7):
-                        print "Dish Mode :-> CONFIG"
-                    elif (self._dish_mode == 8):
-                        print "Dish Mode :-> OPERATE"
-                    else:
-                        print "Dish Mode :-> UNKNOWN!\n", evt
-                except Exception as e:
-                    print "Unexpected error in DishModeCallback!\n", e.message
-            else:
-                print "Error event on subscribing DishMode attribute!\n", evt.errors
-
-    class subarrayObsStateCallback (utils.EventCallback):
-        def push_event(self, evt):
-
-            if (evt.err==False):
-                try:
-                    self._dish_mode = evt.attr_value.value
-                    if(self._dish_mode == 0):
-                        print "Dish Mode :-> OFF"
-                    elif (self._dish_mode == 1):
-                        print "Dish Mode :-> STARTUP"
-                    elif (self._dish_mode == 2):
-                        print "Dish Mode :->  SHUTDOWN"
-                    elif (self._dish_mode == 3):
-                        print "Dish Mode :->  STANDBY-LP"
-                    elif (self._dish_mode == 4):
-                        print "Dish Mode :-> STANDBY-FP"
-                    elif (self._dish_mode == 5):
-                        print "Dish Mode :-> MAINTENANCE"
-                    elif (self._dish_mode == 6):
-                        print "Dish Mode :-> STOW"
-                    elif (self._dish_mode == 7):
-                        print "Dish Mode :-> CONFIG"
-                    elif (self._dish_mode == 8):
-                        print "Dish Mode :-> OPERATE"
-                    else:
-                        print "Dish Mode :-> UNKNOWN!\n", evt
-                except Exception as e:
-                    print "Unexpected error in DishModeCallback!\n", e.message
-            else:
-                print "Error event on subscribing DishMode attribute!\n", evt.errors
-
-
-    class subarrayReceptorIDListCallback (utils.EventCallback):
-        def push_event(self, evt):
+            print CONST.ERR_SUBSR_SA_HEALTH_STATE, evt
+            self._read_activity_message = CONST.ERR_SUBSR_SA_HEALTH_STATE + str(evt)
+            self.devlogmsg(CONST.ERR_SUBSR_SA_HEALTH_STATE, 1)
```

Comment:
Replace 1 with constant name.

Reply:
Replaced magic numbers with respective TANGO Log Levels in Central Node code.

Reply:
Updated log level nos to corresponding enum label for SubarrayNode, DishLeafNode, and DishMaster. Commit id is 8138492.